### ISSUE TITLE
World of Warships: Unable to login using Lutris but works in Steam

### ISSUE THREAD

**Issue by TriMoon** [lutris/lutris#4960]
### Bug description

For a complete history of my journey pls see: https://forum.manjaro.org/t/anyone-able-to-start-and-login-to-world-of-warships-recently/145586

In short:
- It works using Steam + `Proton 5.13-6`
- It doesn't work using Lutris +  `Lutris-GE-Proton8-10-x86_64` (default)
Although "World of Tanks - Blitz" installed in same launcher, does work...

### How to Reproduce

1. Install WGC in  Lutris: https://lutris.net/games/wargaming-game-center/
2. Start and login with your account.
3. Install World of Warships
4. Run the game.
5. Game starts to load and background sounds start playing
6. You will get the usual DxDiag` popup, just dismiss it as usual.
7. It will get stuck at the 'Loging in" stage...

### Expected behavior

Able to still play this game using Lutris, which was possible in past.

### Log output
 `lutris -d | tee /tmp/lutris.terminal-launch.txt`:
- Replaced my `$HOME` with a placeholder using `sed`...
```shell
INFO     2023-08-05 15:04:45,447 [startup.init_lutris:209]:Starting Lutris 0.5.13
INFO     2023-08-05 15:04:45,449 [startup.check_driver:65]:Using NVIDIA drivers 535.86.05 for x86_64
INFO     2023-08-05 15:04:45,449 [startup.check_driver:69]:GPU: NVIDIA GeForce GTX 1080 Ti
INFO     2023-08-05 15:04:45,449 [startup.check_driver:85]:GPU: 10DE:1B06 19DA:4471 (nvidia drivers)
DEBUG    2023-08-05 15:04:45,656 [xrandr._get_vidmodes:16]:Retrieving video modes from XrandR
DEBUG    2023-08-05 15:04:45,732 [lutris.get_missing_game_ids:191]:Checking for missing games
DEBUG    2023-08-05 15:04:54,341 [xrandr._get_vidmodes:16]:Retrieving video modes from XrandR
DEBUG    2023-08-05 15:04:54,347 [command.start:141]:DISABLE_LAYER_AMD_SWITCHABLE_GRAPHICS_1="1"
DEBUG    2023-08-05 15:04:54,347 [command.start:141]:__GL_SHADER_DISK_CACHE="1"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:__GL_SHADER_DISK_CACHE_PATH="/mnt/LutrisGames/wargaming-game-center"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:LD_LIBRARY_PATH="<MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/lib:<MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/lib64:/lib/x86_64-linux-gnu:/lib/i386-linux-gnu:/lib:/usr/lib/x86_64-linux-gnu/libfakeroot:/lib64:/lib32:/lib64:/usr/lib:/usr/lib64:/usr/lib32:/usr/lib64:/usr/lib/i386-linux-gnu:/usr/lib/x86_64-linux-gnu:<MY $HOME>/.local/share/lutris/runtime/Ubuntu-18.04-i686:<MY $HOME>/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:<MY $HOME>/.local/share/lutris/runtime/steam/i386/lib:<MY $HOME>/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:<MY $HOME>/.local/share/lutris/runtime/steam/i386/usr/lib:<MY $HOME>/.local/share/lutris/runtime/Ubuntu-18.04-x86_64:<MY $HOME>/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:<MY $HOME>/.local/share/lutris/runtime/steam/amd64/lib:<MY $HOME>/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:<MY $HOME>/.local/share/lutris/runtime/steam/amd64/usr/lib"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINEDEBUG="-all"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:DXVK_LOG_LEVEL="none"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINEARCH="win64"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINE="<MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/bin/wine"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINE_MONO_CACHE_DIR="<MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/mono"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINE_GECKO_CACHE_DIR="<MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/gecko"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:GST_PLUGIN_SYSTEM_PATH_1_0="<MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/lib64/gstreamer-1.0/:<MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/lib/gstreamer-1.0/"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINEPREFIX="/mnt/LutrisGames/wargaming-game-center"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINEESYNC="1"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINEFSYNC="1"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINE_FULLSCREEN_FSR="1"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:DXVK_NVAPIHACK="0"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:DXVK_ENABLE_NVAPI="1"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:PROTON_BATTLEYE_RUNTIME="<MY $HOME>/.local/share/lutris/runtime/battleye_runtime"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:PROTON_EAC_RUNTIME="<MY $HOME>/.local/share/lutris/runtime/eac_runtime"
DEBUG    2023-08-05 15:04:54,348 [command.start:141]:WINEDLLOVERRIDES="d3d10core,d3d11,d3d12,d3d12core,d3d9,d3dcompiler_33,d3dcompiler_34,d3dcompiler_35,d3dcompiler_36,d3dcompiler_37,d3dcompiler_38,d3dcompiler_39,d3dcompiler_40,d3dcompiler_41,d3dcompiler_42,d3dcompiler_43,d3dcompiler_46,d3dcompiler_47,d3dx10,d3dx10_33,d3dx10_34,d3dx10_35,d3dx10_36,d3dx10_37,d3dx10_38,d3dx10_39,d3dx10_40,d3dx10_41,d3dx10_42,d3dx10_43,d3dx11_42,d3dx11_43,d3dx9_24,d3dx9_25,d3dx9_26,d3dx9_27,d3dx9_28,d3dx9_29,d3dx9_30,d3dx9_31,d3dx9_32,d3dx9_33,d3dx9_34,d3dx9_35,d3dx9_36,d3dx9_37,d3dx9_38,d3dx9_39,d3dx9_40,d3dx9_41,d3dx9_42,d3dx9_43,dxgi,nvapi,nvapi64,nvml=n;winemenubuilder="
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:STEAM_COMPAT_CLIENT_INSTALL_PATH="<MY $HOME>/.local/share/Steam/"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:STEAM_COMPAT_DATA_PATH="/mnt/LutrisGames/wargaming-game-center"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:STEAM_COMPAT_APP_ID="0"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:SteamAppId="0"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:SteamGameId="lutris-game"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:WINE_LARGE_ADDRESS_AWARE="1"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:game_name="Wargaming Game Center"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:PYTHONPATH="/usr/lib/lutris:/usr/games:/usr/lib/python311.zip:/usr/lib/python3.11:/usr/lib/python3.11/lib-dynload:/usr/local/lib/python3.11/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.11/dist-packages"
DEBUG    2023-08-05 15:04:54,349 [command.start:141]:LUTRIS_GAME_UUID="ebb137df-c85a-4e3c-a63f-aebe31875260"
Started initial process 23828 from gamemoderun <MY $HOME>/.local/share/lutris/runners/wine/lutris-GE-Proton8-10-x86_64/bin/wine /mnt/LutrisGames/wargaming-game-center/drive_c/ProgramData/Wargaming.net/GameCenter/wgc.exe
Start monitoring process.
gamemodeauto: 
fsync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
[0805/150504.649:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 0
[0805/150505.694:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 0
[0805/150505.959:INFO:CONSOLE(1)] "[FE][onLoadContent] injectScrollStyles: insert scroll styles to WGC", source:  (1)
[0805/150506.254:INFO:CONSOLE(1)] "[webChannel] connection established.", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.255:INFO:CONSOLE(1)] "%c[React][initAppTransport] Running script with autotests, autotestSource = wgc color: green", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.289:INFO:CONSOLE(1)] "[PostMessageService] init: trustedUrls list [https://worldofwarships.eu,https://worldofwarships.com,https://worldofwarships.asia,https://dailydeals.worldoftanks.eu,https://dailydeals.worldoftanks.com,https://dailydeals.worldoftanks.asia,https://worldoftanks.eu,https://worldoftanks.com,https://worldoftanks.asia,https://worldofwarships.pt,https://surveys.wargaming.net,https://survey.steelhunters.com]", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.289:INFO:CONSOLE(1)] "[PostMessageService] init: start listening messages", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.337:INFO:CONSOLE(1)] "%c[React][AppInstallationMonitor] render: appInstallation is empty; ignore color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.342:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] get contentData: content data is empty; return  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.342:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] get contentData: content data is empty; return  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.349:INFO:CONSOLE(1)] "%c[React][IncludeGameCss] cdm: add body class = wows color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.349:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdm: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.349:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdm: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.350:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdm: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.350:INFO:CONSOLE(1)] "%c[React][GamesMonitor] cdm: selectedGame.tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.350:INFO:CONSOLE(1)] "%c[React][GroupsMonitor] cdm: activeGroupId = 1 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.351:INFO:CONSOLE(1)] "%c[React][GroupsMonitor] cdm: selectedGroup.id = 1 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.351:INFO:CONSOLE(1)] "%c[React][WaitingProvider] cdm: status = 0, message = 'Waiting for response from server...' color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.352:INFO:CONSOLE(1)] "%c[React][AppInstallationMonitor] render: appInstallation is empty; ignore color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.353:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] get contentData: content data is empty; return  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.354:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] get contentData: content data is empty; return  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.354:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.355:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.355:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.355:INFO:CONSOLE(1)] "%c[React][GroupsMonitor] cdu: activeGroupId = 1 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.356:INFO:CONSOLE(1)] "%c[React][GroupsMonitor] cdu: selectedGroup.id = 1, selectedGroup.__id__ = {46ac8811-6c09-4247-be5b-c1d42a04ea38} color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.373:INFO:CONSOLE(1)] "[installedGameCtrl] START construct", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.373:INFO:CONSOLE(1)] "[installedGameCtrl] updateLayout resourcePaths = {"background":"game:/C:/Games/World_of_Warships/game_metadata/resources/background/51b0d9cb0dd2beaf04526d7ce14a528814c5b202_139523.jpg","background_blurred":"game:/C:/Games/World_of_Warships/game_metadata/resources/background_blurred/9130622cbfdbfa1a9e6c565824829f5587f9b903_42213.jpg","big_logo":"","game_css":"game:/C:/Games/World_of_Warships/game_metadata/resources/game_css/81cdf4b3abcc1981e47760d3f9cc4bdefb615c64_1437.css","small_logo":"game:/C:/Games/World_of_Warships/game_metadata/resources/small_logo/5bc40aa01ea74e8ddcd0c4edb6c7103622425c1e_1154.png"}", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.375:INFO:CONSOLE(1)] "[installedGameCtrl] END construct", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.383:INFO:CONSOLE(1)] "[GameStatusDirective] calculateStatusBarType: statusBarType = ready, game.state.starting = 0", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.384:INFO:CONSOLE(1)] "[GameStatusDirective] calculateStatusBarType: statusBarType = ready, game.state.starting = 0", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.404:INFO:CONSOLE(1)] "%c[InstallGameService] onAppInstallationChanged: appInstallation is empty; exit color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.404:INFO:CONSOLE(1)] "[InstallGameService] onErrorChanged: appInstallationError is empty; exit", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.429:INFO:CONSOLE(1)] "[pageContent] START link tag="1527964767"", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.429:INFO:CONSOLE(1)] "[pageContent] updateContentError gameID="WOWS.WW.PRODUCTION" status="NO_ERROR"", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.429:INFO:CONSOLE(1)] "%c[pageContent] onContentOperationInProgressChanged: pageLoadStatus = 0 color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150506.429:INFO:CONSOLE(1)] "[pageContent] END link gameID="WOWS.WW.PRODUCTION"", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.005:INFO:CONSOLE(1)] "%c[React][I18nProvider] updateL11nCache: update l11nCache from qrc://resources/localization/lang_en.json color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.006:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.006:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.008:INFO:CONSOLE(1)] "[app] Running script with autotests", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.026:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] get contentData: content data is empty; return  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.026:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] get contentData: content data is empty; return  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.027:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.027:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.028:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.165:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.166:INFO:CONSOLE(1)] "[pageContent] onContentDataChanged "World of Warships"", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.166:INFO:CONSOLE(1)] "[pageContent] updateContentData gameID="WOWS.WW.PRODUCTION"", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.166:INFO:CONSOLE(1)] "%c[pageContent] updateContentData: pageLoadStatus = 1 color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.167:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.167:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.167:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.168:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.168:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.169:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.312:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.312:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.314:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.314:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150507.314:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150509.300:INFO:CONSOLE(1)] "[GameStatusDirective] calculateStatusBarType: statusBarType = ready, game.state.starting = 0", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150509.302:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150509.302:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150509.302:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.086:INFO:CONSOLE(1)] "[GameStatusDirective] calculateStatusBarType: statusBarType = ready, game.state.starting = 0", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.086:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.087:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.087:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.405:INFO:CONSOLE(1)] "[GameStatusDirective] calculateStatusBarType: statusBarType = running, game.state.starting = 0", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.407:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.407:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150511.407:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150607.262:INFO:CONSOLE(1)] "[GameStatusDirective] calculateStatusBarType: statusBarType = ready, game.state.starting = 0", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150607.264:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 0, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150607.264:INFO:CONSOLE(1)] "%c[React][ShopItemsMonitor] cdu: collectionId = 1, selectedItemIndex = null, enableStorefront = undefined,  color: #cecece", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
[0805/150607.264:INFO:CONSOLE(1)] "%c[React][SelectedGameWrapper] cdu: selected game appId = WOWS.WW.PRODUCTION, tag = 1527964767 color: darkcyan", source: qrc:/ui/wgc.ae6c40950f204e5b39ae.js (1)
wine client error:294: write: Bad file descriptor
wine client error:1c8: write: Bad file descriptor
wine client error:220: write: Bad file descriptor
Initial process has exited (return code: 0)
Monitored process exited.
Exit with return code 0
DEBUG    2023-08-05 15:06:16,908 [command.on_stop:195]:Process 23826 has terminated with code 0
DEBUG    2023-08-05 15:06:18,440 [game.beat:842]:Game thread stopped
WARNING  2023-08-05 15:06:18,440 [game.on_game_quit:879]:Game still running (state: running)
INFO     2023-08-05 15:06:18,441 [game.stop:853]:Stopping Wargaming Game Center (wine)
DEBUG    2023-08-05 15:06:18,441 [game.stop_game:809]:Wargaming Game Center (wine) has run for 84 seconds
DEBUG    2023-08-05 15:06:18,442 [application.on_game_stop:679]:Removing 1 from running IDs
DEBUG    2023-08-05 15:06:18,488 [game.stop_game:821]:Playtime: 184 hours and 16 minutes
DEBUG    2023-08-05 15:06:18,488 [game.on_game_quit:897]:Wargaming Game Center stopped at Sat, 05 Aug 2023 15:06:18
DEBUG    2023-08-05 15:06:18,527 [lutris.add_to_path_cache:158]:Adding Wargaming Game Center (wine) to path cache
2023-08-05 15:07:19,022: Starting Lutris 0.5.13
2023-08-05 15:07:19,024: Using NVIDIA drivers 535.86.05 for x86_64
2023-08-05 15:07:19,025: GPU: NVIDIA GeForce GTX 1080 Ti
2023-08-05 15:07:19,025: GPU: 10DE:1B06 19DA:4471 (nvidia drivers)
2023-08-05 15:07:24,346: Shutting down Lutris
```

### System Information

```shell
[System]
OS:              Ubuntu 23.04 Lunar Lobster
Arch:            x86_64
Kernel:          6.2.0-26-generic
Desktop:         KDE
Display Server:  wayland

[CPU]
Vendor:          GenuineIntel
Model:           Intel(R) Core(TM) i7-5820K CPU @ 3.30GHz
Physical cores:  6
Logical cores:   12

[Memory]
RAM:             31.2 GB
Swap:            0.0 GB

[Graphics]
Vendor:          NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce GTX 1080 Ti/PCIe/SSE2
OpenGL Version:  4.6.0 NVIDIA 535.86.05
OpenGL Core:     4.6.0 NVIDIA 535.86.05
OpenGL ES:       OpenGL ES 3.2 NVIDIA 535.86.05
Vulkan Version:  1.3.239
Vulkan Drivers:  NVIDIA GeForce GTX 1080 Ti (1.3.242)
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by TriMoon** [comment]
@H34dl3ss 
> I would be glad to support some logs if necessary, but I need some guidance how to grab them.

I literally posted the command used to grab the logs i posted in my post :wink:

**Comment by H34dl3ss** [comment]
Thanks @TriMoon that will be useful, but at the moment it seems like no one is taking care of this issue at all.

**Comment by TriMoon** [comment]
@H34dl3ss seems so, but let's be patience who knows :wink:

**Comment by H34dl3ss** [comment]
With latest Lutris update 0.5.14 and therefore delivered wine-ge-8-21-x86_64 the issue is gone.

**Comment by GloriousEggroll** [comment]
it wasnt a problem with lutris, it was a problem with wine:

```
[GE-Proton8-20 Released](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton8-20)

HOTFIX:

-updated world of warships workaround patch with proper fix, no longer requires environment variable (thanks OOOOOF123)
```
https://github.com/ValveSoftware/wine/pull/201

Closing as resolved.

**Comment by TriMoon** [comment]
Because [GE-Proton8-20 Released](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton8-20) is still not available on `Kubuntu 23.10`, i have switched to `wine-ge-8-22` _(which was available)_ and all seems to work.
:+1:

**Comment by Kyr4l** [comment]
this is still an issue
